pkg/trtl/honu_test.go

```diff
@@ -160,11 +255,8 @@ func (s *trtlTestSuite) TestBatch() {
 	require.Equal(int64(len(requests)), reply.Failed)
 	require.Equal(int64(0), reply.Successful)
 	require.Len(reply.Errors, len(requests))
-	for _, e := range reply.Errors {
-		require.Contains(requests, e.Id)
-		require.Equal(requests[e.Id].Id, e.Id)
-		require.Contains(e.Error, "not implemented")
-	}
+	require.Contains(requests, reply.Errors[1].Id)
+	require.Equal(requests[reply.Errors[1].Id].Id, reply.Errors[1].Id)
```

Comment:
FYI - had to update this test now that Put is implemented, but I can do a better update when I implement Delete in my next PR